src/app.rs

```diff
@@ -460,10 +462,11 @@ pub fn build_app() -> App<'static, 'static> {
                 .long_help(
                     "Filter results based on the file modification time. The argument can be provided \
                      as a specific point in time (YYYY-MM-DD HH:MM:SS) or as a duration (10h, 1d, 35min). \
-                     '--change-older-than' can be used as an alias.\n\
+                     '--change-older-than' or '--older' can be used as an alias.\n\
```

Comment:
maybe here also "can be used as aliases", for consistency. Thank you for the updates!

Reply:
Ofc np, just pushed up the changes.